cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -21,7 +21,14 @@
     {%- endfor %}
 
     {% for block in page.content -%}
-        {{ render_block.render(block, loop.index) }}
+        {% if block.block_type == 'feedback' %}
```

Comment:
I think a full solution for this is out of scope for these PR's. My recommendation for it would be to create a template that would be imported/included that would encompass all of if/else clauses the pages have. 

I can make the change in a different PR, unless it doesn't seem as out of scope as I'm suggesting.